lib/elixir_analyzer/exercise_test/feature.ex

```diff
@@ -53,10 +53,16 @@ defmodule ElixirAnalyzer.ExerciseTest.Feature do
     feature_data = %{feature_data | meta: Map.to_list(feature_data.meta)}
     feature_data = Map.to_list(feature_data)
 
+    unless Keyword.has_key?(feature_data, :comment) do
+      raise "Comment must be defined for each feature test"
+    end
```

Comment:
Awesome 🎉  I didn't notice before that this one macro is missing the comment check